ui/widgets/scalarparameterwidget.cpp

```diff
@@ -226,6 +228,8 @@ void ScalarParameterWidget::getParameterFromGUI(){
         QLineEdit *doubleValue = static_cast<QLineEdit*>(doubleIterator.value());
         if(doubleValue->hasAcceptableInput()){ //if line edit contains a double
             this->doubleParameter.insert(doubleKey, doubleValue->text().toDouble());
+        } else if (doubleValue->text().trimmed() == "") {
```

Comment:
```suggestion
        } else if (doubleValue->text().isEmpty()) {
```

https://doc.qt.io/qt-5/qstring.html#isEmpty